src/Compilers/Server/VBCSCompiler/ServerDispatcher.cs

```diff
@@ -246,10 +233,18 @@ private void HandleCompletedGCTask()
             GC.Collect();
         }
 
+        private void MaybeCreateListenTask()
+        {
+            if (_listenTask is null && _clientConnectionHost.IsListening)
```

Comment:
>_clientConnectionHost.IsListening [](start = 39, length = 33)

When is this false?

Reply:
Good catch: second part of the condition is always true